installer/dbscript/dbscript-1.sql

```diff
@@ -2352,3 +2352,16 @@ ALTER TABLE `ohrm_i18n_lang_string`
 
 ALTER TABLE `ohrm_i18n_translate`
     ADD CONSTRAINT `translateUniqueId` UNIQUE (`lang_string_id`, `language_id`);
+
+CREATE TABLE `ohrm_mail_queue` (

```

Comment:
shall we move this to `dbscript-2.sql` to track 5.x migration

Reply:
movd